packages/wallet-sdk/src/provider/CoinbaseWalletProvider.ts

```diff
@@ -646,7 +628,7 @@ export class CoinbaseWalletProvider
     return response;
   }
 
-  private _setAddresses(addresses: string[], isDisconnect?: boolean): void {
+  protected _setAddresses(addresses: string[], _?: boolean): void {
```

Comment:
isDisconnect is only used for walletlinked connected extension. moving this to extension provider subclass